src/adapter/src/coord.rs

```diff
@@ -4264,11 +4353,7 @@ impl<S: Append + 'static> Coordinator<S> {
     }
 
     /// The smallest common valid read frontier among the specified collections.
-    fn least_valid_read(
-        &self,
-        id_bundle: &CollectionIdBundle,
-        instance: ComputeInstanceId,
-    ) -> Antichain<mz_repr::Timestamp> {
+    fn least_valid_read(&self, id_bundle: &CollectionIdBundle) -> Antichain<mz_repr::Timestamp> {
```

Comment:
This feels like it might be an overall correctness improvement because the timestamp selection is not unrelated to your current compute instance? I think that's good?

Reply:
I'm having a little trouble following the double negatives in this comment. In theory I think that timestamp selection is independent of the current compute instance, since strict serializablity should cross compute instance boundaries. For example if you perform a read at `ts` in compute instance `foo`. Then all later reads in compute instance `bar` should be at `ts` or higher.

In practice I don't think it's possible for a user query to contain multiple compute instances.